library/experimental/mutation/pod-security-policy/capabilities/samples/mutation-modifyset.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: mutations.gatekeeper.sh/v1alpha1
+kind: ModifySet
+metadata:
+  name: k8spspcapabilities
+spec:
+  applyTo:
+    - groups: [""]
+      versions: ["v1"]
+      kinds: ["Pod"]
+  match:
+    scope: Namespaced
+    kinds:
+      - apiGroups: [""]
+        kinds: ["Pod"]
+  location: "spec.containers[name:*].securityContext.capabilities.add"
+  parameters:
+    values:
+      operation: merge
```

Comment:
I think `operation` may be in the wrong place?

Reply:
Thanks! fixed